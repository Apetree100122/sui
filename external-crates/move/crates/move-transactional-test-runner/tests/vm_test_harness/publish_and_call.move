//# publish-and-call --call 0x42::m::a --call 0x42::m::b
module 0x42::m {
    fun a(): u64 {
        0
    }

    fun b(): u64 {
        0
    }
}
