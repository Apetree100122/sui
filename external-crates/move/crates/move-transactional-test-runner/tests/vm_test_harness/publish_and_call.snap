---
source: crates/move-transactional-test-runner/src/framework.rs
---
processed 1 task

task 0, lines 1-10:
//# publish-and-call --call 0x42::m::a --call 0x42::m::b
return values: 0
return values: 0
